automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoModelCopy.scala

```diff
@@ -54,7 +55,8 @@ case class DefaultLabels(clusterName: ClusterName,
                          clusterServiceAccount: Option[WorkbenchEmail],
                          notebookServiceAccount: Option[WorkbenchEmail],
                          notebookExtension: Option[String],
-                         notebookUserScript: Option[String]) {
+                         notebookUserScript: Option[String],
+                         jupyterUserScriptUri: Option[String]) {
```

Comment:
[drive by comment without real understanding]

Do you mean `jupyterStartUserScriptUri` here?

Reply:
Good catch - this was wrong. I think to match the convention here I want `notebookStartUserScript`; unclear to me why this is different than the rest of the naming.